src/client.cpp

```diff
@@ -340,32 +339,12 @@ PVR_ERROR GetChannels(ADDON_HANDLE handle, bool bRadio)
   return PVR_ERROR_SERVER_ERROR;
 }
 
-bool OpenLiveStream(const PVR_CHANNEL &channel)
```

Comment:
Why have you replaced OpenLiveStream by an empty implementation? Is it not needed if a stream url is provided (was the case before, just a little different), means this was nonsense even before the latest API changes? Sorry for the dumb question but me not being deeply into the streaming details is part of my story telling people that fixing all addons is not an easy (10 minutes, haha!) task for me.